### ISSUE TITLE
Error: fields must be passed when initializing state

### ISSUE THREAD

**Issue by plexigras** [erikras/redux-form#2410]
```jsx
import React, { PropTypes } from 'react';
import { Field, reduxForm } from 'redux-form';

const Searchbar = ({ handleSubmit }) => (
  <form onSubmit={handleSubmit}>
    <Field name="q" component="input" type="text"/>
    <button type="submit" >Search</button>
  </form>
);

Searchbar.displayName = 'Searchbar';
Searchbar.propTypes = {
  handleSubmit: PropTypes.func
};

export default reduxForm({
  form: 'search'
})(Searchbar);
```

**Comment by gustavohenke** [comment]
Saw no problems at all with what I thought was the minimum reproducible code: http://www.webpackbin.com/NyOCeUhSM

Can you elaborate? Include version numbers, and if possible enhance the above webpackbin with whatever you got in order to reproduce your issue.

**Comment by plexigras** [comment]
thx. i installed the wrong version.

**Comment by samjbmason** [comment]
Out of curiosity @plexigras  what version did you have installed? We have started getting this error with `5.2.5`

**Comment by plexigras** [comment]
5.3.4 which i believe is or at least was the default provided by npm but the documentation was for 6.4.3

http://redux-form.com just redirected me straight to http://redux-form.com/6.4.3

**Comment by gustavohenke** [comment]
This is some mess related to the latest releases.
Some fixes for people still using v5 were published as v5.3.4 after v6.4.3 was released.

I think @erikras recently tagged v6.4.3 as `latest` to avoid further problems with this.

**Comment by thom-nic** [comment]
I encountered this problem as of v5.3.4 up to v5.3.6, rolling back to v5.3.3 fixed it.
